giraffe/src/types/index.ts

```diff
@@ -453,6 +455,12 @@ export interface ScatterLayerConfig {
   symbol?: string[]
 }
 
+export interface SimpleTableLayerConfig {
+  type: 'simple table' // do not refactor or restrict to LayerTypes.Mosaic
```

Comment:
```suggestion
  type: 'simple table' // do not refactor or restrict to LayerTypes.SimpleTable
```
@TCL735 what do these comments even mean?